target/scripts/helpers/index.sh

```diff
@@ -1,16 +1,25 @@
 #! /bin/bash
+
 # shellcheck source-path=target/scripts/helpers
 # This file serves as a single import for all helpers
 
+export CHKSUM_FILE=/tmp/docker-mailserver-config-chksum
+
 function _import_scripts
 {
   local PATH_TO_SCRIPTS='/usr/local/bin/helpers'
 
-  . "${PATH_TO_SCRIPTS}/postfix.sh"
-  . "${PATH_TO_SCRIPTS}/accounts.sh"
-  . "${PATH_TO_SCRIPTS}/aliases.sh"
-  . "${PATH_TO_SCRIPTS}/relay.sh"
-  . "${PATH_TO_SCRIPTS}/sasl.sh"
-  . "${PATH_TO_SCRIPTS}/ssl.sh"
+  source "${PATH_TO_SCRIPTS}/accounts.sh"
+  source "${PATH_TO_SCRIPTS}/aliases.sh"
+  source "${PATH_TO_SCRIPTS}/dns.sh"
+  source "${PATH_TO_SCRIPTS}/exit.sh"
+  source "${PATH_TO_SCRIPTS}/log.sh"
+  source "${PATH_TO_SCRIPTS}/miscellaneous.sh"
```

Comment:
What do you think about just `misc.sh`?

Reply:
The file itself could presently also be split to two separate ones `lock.sh` and `utils.sh`

Reply:
@casperklein would you agree to further split this, as @polarathene suggested, into `lock.sh` and `utils.sh`? I would then go ahead and do this. If not, I will rename it to `misc.sh`.

Reply:
> would you agree to further split this, as @polarathene suggested

I'll leave it up to you both. Just keep in mind, that endless splitting will maybe have the opposite effect you wanted to solve with this PR. The misc file wasn't that big and confusing that I would have split it.

Reply:
Alright. I went ahead and split it, because I assume the lock logic could change with future PRs, so I figured this was a good thing to do in this case. But I agree: endless splitting is not a good idea and we'll need to take care in the future.